pcsx2-qt/Settings/AchievementSettingsWidget.ui

```diff
@@ -49,7 +49,7 @@
       <item row="0" column="1">
        <widget class="QCheckBox" name="richPresence">
         <property name="text">
-         <string>Enable Rich Presence</string>
+         <string>Enable RA's Rich Presence</string>
```

Comment:
Are you sure that this will not just confuse people?

Reply:
I'm not sure, but I do know that not telling what Rich Presence this is if there's actually two Rich Presences is also confusing. Maybe we could change this like the Discord version and say "Enable RetroAchievements' Presence"?